frontend/src/CommitInfo.res

```diff
@@ -40,14 +40,18 @@ let renderExternalLink = (~href, text) => {
   <a target="_blank" className={Sx.make(sx)} href> {Rx.text(text)} </a>
 }
 
+let url: string = %raw(`import.meta.env.VITE_PIPELINE_FRONTEND_URL`)
+
+let port: string = %raw(`import.meta.env.VITE_PIPELINE_PORT`)
+
 let renderJobIdLink = jobId => {
   let shortJobId = switch String.split_on_char('/', jobId) {
   | list{_, shortJobId} => shortJobId
   | _ =>
     Js.log(("Error: invalid jobId", jobId))
     jobId
   }
-  let href = "http://autumn.ocamllabs.io:8081/job/" ++ jobId
+  let href = url ++ ":" ++ port ++ "/job/" ++ jobId
```

Comment:
As mentioned above `host` should be used instead of `url` here. In which case we need to preserve `http`.

This is unfortunate because we're likely to switch to `https` in the future, while still using `http` in development.

I suggest to hardcode `http` here for now (as it will work just fine before we introduce HTTPS). We can add an additional var to device what environment we're using (prod or dev).